src/features/user/components/UserDropDown.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+
+import Link from "next/link"
+import { Button } from "@/components/ui/button"
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from "@/components/ui/dropdown-menu"
+
+export function UserDropDown() {
+  const handleLogout = () => {
+    console.log("logout")
+  }
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <Button variant="ghost">
+          山田 花子
+        </Button>
```

Comment:
Buttonのvariantは"secondary"に修正をお願いします！

```suggestion
        <Button variant="secondary">
          山田 花子
        </Button>
```